rustic-cargo.el

```diff
@@ -446,6 +446,22 @@ If running with prefix command `C-u', read whole command from minibuffer."
                       (concat "cargo add " (read-from-minibuffer "Crate: ")))))
       (rustic-run-cargo-command command))))
 
+(defun rustic-cargo-add-missing-imports ()
+  "Add missing imports to Cargo.toml.
+Adds all missing imports by default.
+Use with 'prefix-arg` to select imports to add."
+  (interactive)
+  (let ((missing-imports (delete-dups
+                          (seq-reduce (lambda (missing-crates errortable)
+                                        (if (string= "E0432" (gethash "code" errortable))
+                                            (cons (nth 3 (split-string (gethash "message" errortable) "`")) missing-crates)
+                                          missing-crates))
+                                      (gethash (buffer-file-name) (lsp-diagnostics t)) '()))))
+    (rustic-run-cargo-command (format  "cargo add %s"
```

Comment:
Why not use `rustic-cargo-add` ?

Reply:
`rustic-cargo-add` reads from the minibuffer, so I figured it was not meant for the case that's needed here. Is there a way to use `rustic-cargo-add` that I am missing? Of course, `rustic-cargo-add` could be modified to work with `rustic-cargo-add-missing-import`, but I think the end result would be worse than the current situation.

Reply:
You're right, I haven't thought about it.

I think we should also handle the case when `missing-imports` is `nil`.

Reply:
Done!